folders/backend/express-postgres/server.ts

```diff
@@ -0,0 +1,109 @@
+import express from 'express';
+import { ClientError, errorMiddleware } from './lib/index.js';
+
+import pg from 'pg';
+
+pg.types.setTypeParser(pg.types.builtins.NUMERIC, parseFloat);
```

Comment:
Nice, but for future reference, it's only necessary if you are querying columns that are floating point (non-integer numbers).